### ISSUE TITLE
Crash: Settings key bluetooth_address only readable with targetSdkVersion <= 31

### ISSUE THREAD

**Issue by Mrnofish** [kamgurgul/cpu-info#81]
The crash can be reliably reproduced by tapping on the Hardware and Sensors tabs.

LOS 20 / Android 13.

```
FATAL EXCEPTION: main
Process: com.kgurgul.cpuinfo, PID: 3237
java.lang.SecurityException: Settings key: <bluetooth_address> is only readable to apps with targetSdkVersion lower than or equal to: 31
at android.provider.Settings$NameValueCache.getStringForUser(Settings.java:3081)
at android.provider.Settings$Secure.getStringForUser(Settings.java:6217)
at android.provider.Settings$Secure.getString(Settings.java:6173)
at com.kgurgul.cpuinfo.features.information.hardware.HardwareInfoViewModel.p(SourceFile:271)
at com.kgurgul.cpuinfo.features.information.hardware.HardwareInfoViewModel.s(SourceFile:115)
at com.kgurgul.cpuinfo.features.information.hardware.HardwareInfoViewModel.<init>(SourceFile:77)
at p4.k$m$a.get(SourceFile:330)
at t5.d$a.e(SourceFile:44)
at androidx.lifecycle.a.b(SourceFile:34)
at t5.d.b(SourceFile:15)
at androidx.lifecycle.s0.b(SourceFile:61)
at androidx.lifecycle.s0.a(SourceFile:29)
at androidx.lifecycle.r0.a(SourceFile:40)
at androidx.lifecycle.r0.getValue(Unknown Source:0)
at b5.b.f2(Unknown Source:2)
at b5.b.d2(SourceFile:3)
at com.kgurgul.cpuinfo.features.information.base.b.E0(SourceFile:49)
at androidx.fragment.app.Fragment.h1(SourceFile:20)
at androidx.fragment.app.c0.f(SourceFile:199)
at androidx.fragment.app.c0.m(SourceFile:125)
at androidx.fragment.app.w.d0(SourceFile:231)
at androidx.fragment.app.w.j1(SourceFile:82)
at androidx.fragment.app.w.b0(SourceFile:32)
at androidx.fragment.app.a.j(SourceFile:7)
at androidx.viewpager2.adapter.FragmentStateAdapter.X(SourceFile:130)
at androidx.viewpager2.adapter.FragmentStateAdapter.U(SourceFile:1)
at androidx.viewpager2.adapter.FragmentStateAdapter.C(Unknown Source:2)
at androidx.recyclerview.widget.RecyclerView.F(SourceFile:14)
at androidx.recyclerview.widget.RecyclerView$e.f(SourceFile:8)
at androidx.recyclerview.widget.d.a(SourceFile:26)
at androidx.recyclerview.widget.RecyclerView$p.f(SourceFile:124)
at androidx.recyclerview.widget.RecyclerView$p.e(Unknown Source:1)
at androidx.recyclerview.widget.RecyclerView$p.d(Unknown Source:1)
at androidx.recyclerview.widget.LinearLayoutManager.u2(SourceFile:35)
at androidx.recyclerview.widget.LinearLayoutManager.Z1(SourceFile:39)
at androidx.recyclerview.widget.LinearLayoutManager.D2(SourceFile:35)
at androidx.recyclerview.widget.LinearLayoutManager.A1(SourceFile:8)
at androidx.recyclerview.widget.RecyclerView.s1(SourceFile:26)
at androidx.recyclerview.widget.RecyclerView$a0.j(SourceFile:59)
at androidx.recyclerview.widget.RecyclerView$e0.run(SourceFile:385)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1231)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1239)
at android.view.Choreographer.doCallbacks(Choreographer.java:899)
at android.view.Choreographer.doFrame(Choreographer.java:827)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1214)
at android.os.Handler.handleCallback(Handler.java:942)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loopOnce(Looper.java:201)
at android.os.Looper.loop(Looper.java:288)
at android.app.ActivityThread.main(ActivityThread.java:7884)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by kamgurgul** [comment]
This issue is already fixed in version `4.7.0`

**Comment by Mrnofish** [comment]
Apologies for the bogus report, I have no idea why Neo Store didn't signal the 4.7 upgrade, despite it having been published over a week ago.
